kube/charts/helm.go

```diff
@@ -16,30 +16,28 @@
    limitations under the License.
 */
 
-package helm
+package charts
```

Comment:
Actually, I'd prefer to keep "helm" as the package name (but it makes sense to remove the subfolder charts), "helm" is a more understandable as a kube sub-folder / package